Makefile

```diff
@@ -32,10 +32,35 @@ dependencies-ocaml-with-z3:
 dependencies-js:
 	$(MAKE) -C $(FRENCH_LAW_JS_LIB_DIR) dependencies
 
+PY_VENV_DIR = _python_venv
+
+PY_VENV_ACTIVATE = . $(PY_VENV_DIR)/bin/activate;
+
+# Rebuild when requirements change
+$(PY_VENV_DIR): $(PY_VENV_DIR)/stamp
+
+$(PY_VENV_DIR)/stamp: \
+    runtimes/python/catala/pyproject.toml \
+    syntax_highlighting/en/pygments/pyproject.toml \
+    syntax_highlighting/fr/pygments/pyproject.toml \
+    syntax_highlighting/pl/pygments/pyproject.toml
+	test -d $(PY_VENV_DIR) || python3 -m venv $(PY_VENV_DIR)
+	$(PY_VENV_ACTIVATE) python3 -m pip install -U \
+	  -e runtimes/python/catala \
+	  -e syntax_highlighting/en/pygments \
+	  -e syntax_highlighting/fr/pygments \
+	  -e syntax_highlighting/pl/pygments
+	touch $@
+
+# Run sub-make within the Python venv
+MAKEP = $(PY_VENV_ACTIVATE) $(MAKE)
```

Comment:
Yeah that's a pain. We really need to tell in the `README` to people to use the `Makefiles` to get literate outputs.